### PR TITLE
[Chore](Job)Add the configuration of the maximum number of persistence tasks for the job

### PR DESCRIPTION
**Pull request by CalvinKirs**
…

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1585,7 +1585,10 @@ public class Config extends ConfigBase {
      */
     @ConfField(description = {"任务堆积时用于存放定时任务的队列大小", "The number of timer jobs that can be queued."})
     public static int job_dispatch_timer_job_queue_size = 1024;
-
+    @ConfField(description = {"一个 Job 的 task 最大的持久化数量，超过这个限制将会丢弃旧的 task 记录, 如果值 < 1, 将不会持久化。",
+            "Maximum number of persistence allowed per task in a job,exceeding which old tasks will be discarded，"
+                   + "If the value is less than 1, it will not be persisted." })
+    public static int max_persistence_task_count = 100;
```
Comment (zddr) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1591]: what will happen,if set to -1

**Comment by CalvinKirs** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java (hunk 1 above)
Comment (CalvinKirs) [reply fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1591]: If the value is less than 1, it will not be persisted.

**Comment by CalvinKirs** [comment]
If persistence will be unified in the job later, I think it might be better to add a new method of persisting association records.

**Review by morningman — APPROVED** [review]
LGTM

**Comment by CalvinKirs** [comment]
run feut

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**